CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### Features
 
+- v3.3.0
```

Comment:
Lets not bump the version here. Will do that and add the changelog as I make a new release.